src/components/page-pipeline-jobs/style.css

```
@@ -7,3 +7,19 @@
 .pipeline-job-action__action--justify-end {
   justify-self: end;
 }
+
+.application-alerting-content {
+  padding: var(--eds_spacing_medium);
+  padding-top: 0;
+  overflow-y: auto;
+}
+@media (min-width: 20rem) {
+  .application-alerting-content {
+    width: 18rem;
+  }
+}
+@media (min-width: 50rem) {
+  .application-alerting-content {
+    width: 25rem;
```

I would suggest increasing the width of the scrim a little bit on larger screens.
```suggestion
    width: 28rem;
```

---

fixed